### ISSUE TITLE
alias request for game in development

### ISSUE THREAD

**Issue by nolorfin** [Humblemonk/DiceMaiden#115]
I'd like to request an alias for a game system we are currently playtesting.

Rolls can be made with 1 to 10 dice: either D6, which go from 0 to 5, or D10, which go from 0-9. The highest two results are counted. Each maximum die result (either 5s or 9s, depending on the die rolled) allows counting an additional die. A bonus from -5 to +5 is added to the total.

The system has more nuts and bolts, but that would actually be enough. We are currently using common D6s, and many players have difficulty remembering they must count 6s as 0s.

If more details are needed, please don't hesitate to ask.

Thanks.

**Comment by Humblemonk** [comment]
Was this game system ever finalized?

**Comment by nolorfin** [comment]
Yes, we played/tested it for a couple of years, but then the group disbanded. I now play with two other groups, but haven't introduced them to the game. I am currently satisfied with what I get from existing game engines.

There are a few things I would tweak about the system, but it worked.
